docs/mkdocs/docs/features/assertions.md

```diff
@@ -103,29 +103,37 @@ behavior and yields a runtime assertion.
     Assertion failed: (m_object != nullptr), function operator++, file iter_impl.hpp, line 368.
     ```
 
-### Reading from a null `FILE` pointer
+## Changes
 
-Reading from a null `#!cpp FILE` pointer is undefined behavior and yields a runtime assertion. This can happen when
-calling `#!cpp std::fopen` on a nonexistent file.
+### Reading from a null `FILE` or `char` pointer
 
-??? example "Example 4: Uninitialized iterator"
+Reading from a null `#!cpp FILE` or `#!cpp char` pointer is undefined behavior.
+Until version 3.11.4, a runtime assertion was triggered.
+Since version 3.11.4, a [`parse_error.101`](../home/exceptions.md#jsonexceptionparse_error101) is thrown instead.
```

Comment:
Something like this?

Reading from a null `#!cpp FILE` or `#!cpp char` pointer in C++ is undefined behavior.
Until version 3.11.4, this library asserted that the pointer was not `nullptr` using a runtime assertion.  If assertions were disabled, this would result in undefined behavior.
Since version 3.11.4, this library checks for `nullptr` and throws a [`parse_error.101`](../home/exceptions.md#jsonexceptionparse_error101) to prevent the undefined behavior.